src/Logbook/ILogbook.sol

```diff
@@ -0,0 +1,91 @@
+//SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "./IPaymentSplitter.sol";
+
+interface ILogbook is IPaymentSplitter, IERC721 {
+    event SetTitle(uint256 indexed tokenId, string title);
+    event SetDescription(uint256 indexed tokenId, string description);
+    event SetForkPrice(uint256 indexed tokenId, uint256 forkPrice);
+    event Publish(
+        uint256 indexed tokenId,
+        uint256 indexed logContentHash,
```

Comment:
If we want to generalize `Publish` event, shall we name `logContentHash` simply as `contentHash`? Since other publish events might not have the concept of log.